src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/BotSegments/Models/CVObjectDetectionMouseActionData.cs

```diff
@@ -138,7 +138,7 @@ private void RequestCVObjectDetectionTextEvaluation(int segmentNumber)
                                     data = screenshot
                                 },
                                 textQuery: textQuery,
-                                imageQuery: null,
+                                imageQuery: imageQuery,
```

Comment:
Similar / same comment as
https://github.com/Regression-Games/RGUnityBots/pull/266/files#r1732779487

It feels strange to allow the user to pass both queries, but then prefer the text query endpoint.  I feel like these need to be 2 separate criteria to avoid this potential user configuration error.